lib/geometry/arc.rb

```diff
@@ -13,12 +13,17 @@ module Geometry
     arc = Geometry::Arc.new center:[1,1], radius:2, start:0, end:90
 =end
 
-    class Arc
+class Arc
 	include ClusterFactory
 
 	attr_reader :center
 	attr_reader :radius
 	attr_reader :start_angle, :end_angle
+	attr_writer :options
+	def options
+		@options = {} if !@options
+		@options
```

Comment:
Why not:
```ruby
def options
  @options ||= {}
end
```

Reply:
For sure, I had just copied pasted and should have reviewed the code itself for improvements